rmf_fleet_adapter/src/rmf_fleet_adapter/tasks/Patrol.cpp

```diff
@@ -67,8 +69,20 @@ void add_patrol(
             std::make_move_iterator(place.errors.begin()),
             std::make_move_iterator(place.errors.end()));
         }
-        auto desc = GoToPlace::Description::make_with_multiple(goals);
+        auto desc = GoToPlace::Description::make_for_one_of(goals);
         return {desc, errors};
+
+        const auto constraints = msg.find("constraints");
+        if (constraints != msg.end())
+        {
+          for (const auto& constraint : constraints.value())
+          {
+            if (constraint["type"].get<std::string>() == "prefer_same_map")
```

Comment:
Constraint property should be `category` instead of `type` according to the schema

Reply:
I can't believe I messed up my own schema.